src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -98,7 +102,7 @@ RoboCup Bangkok was the first in-person competition in three years due to COVID-
 #### Hardware
 
 - Find a replacement for CM740’s quick so we can buy or make them before the end of the year.
-  - Redesign NUsense to only contain the core functionality – then additional functionality can be added with shields – modular system – modules that you can plug in
+  - Redesign NUsense to only contain the core functionality. Additional functionality can be added with daughterboards/shields, creating a more modular system.
 - Get a decent battery powered soldering iron (TS100), a decent stand, sponge, and safety glasses with some back up wire and various connections
```

Comment:
I have seen some interesting battery-powered stuff from ryobi. You might want to look into their range and see if that meets your needs (this may also lead into using ryobi batteries for the robots --- use the same battery for everything)